EIPS/eip-2381.md

```diff
@@ -0,0 +1,143 @@
+---
+eip: 2381
+title: ENS support for EIP-721 TokenID
+author: Oisin Kyne (@OisinKyne)
+discussions-to: https://github.com/ethereum/EIPs/issues/2381
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-11-15
+requires: 137, 165, 721
+---
+
+## Simple Summary
+
+This EIP makes [EIP-721](https://eips.ethereum.org/EIPS/eip-721) Non-Fungible Tokens addressable by the Ethereum Name Service ([EIP-137](https://eips.ethereum.org/EIPS/eip-137)) by adding a new resolver profile, `uint256 tokenID()`. This allows the Ethereum Name Service to address a single non-fungible token within an EIP-721 token contract of many tokens.
+
+## Abstract
+
+It makes sense to resolve an ENS name to just a contract address for fungible tokens such as [EIP-20](https://eips.ethereum.org/EIPS/eip-20), as each token in the contract is indistinguishable from another. However, for non-fungible tokens; pointing to the contract address alone is not enough, as tokens within the contract are meant to be unique and distinguishable. Non-fungibles might have different valuations, might grant the holder different rights or rewards etc.
+
+A single deployed EIP-721 contract can contain multiple non-fungible tokens. Tokens are addressed by an unsigned 256bit integer, `tokenID`. This EIP describes an extra getter and setter, `tokenID()` and `setTokenID()`, that are included in an ENS resolver contract, to allow for ENS names to resolve to a specific non fungible token within a given EIP-721 contract.
+
+This allows ENS domains like [`bugcat.cryptokitties.eth`](https://www.cryptokitties.co/kitty/101) and [`dragon.cryptokitties.eth`](https://www.cryptokitties.co/kitty/896775) to resolve to both the CryptoKitties contract address and the tokenID of the non-fungible kitty within it.
```

Comment:
This content all seems like motivation.  Abstract should be a human readable version of the core of the technical specification.  Something along the lines of:
> Creates a standard ENS resolver interface that contains the methods `tokenID(node)` and `setTokenID(node, token)`.  These methods allow an ENS user to set/query...

Reply:
Moved the non technical aspects to motivation